### DISCUSSION TITLE
Looking at a product spec sheet, how do I tell if a device supports SPIR-V?

### DISCUSSION THREAD

**Discussion by ArcherSmythe** [CHIP-SPV/chipStar#648] [General]
If I'm trying to decide what piece of accelerated hardware to buy, how can I tell if a given accelerator supports SPIR-V? Basically, just trying to avoid the situation that arose [in this thread](https://github.com/CHIP-SPV/chipStar/discussions/619), where NVIDIA supports OpenCL but not SPIR-V.

**Comment by pjaaskel** [comment]
To my knowledge Intel GPUs and some Arm Mali mobile GPUs are the only ones that officially support SPIR-V at the moment. We have added initial experimental SPIR-V support for NVIDIA GPUs in the PoCL-CUDA driver. Also RustiCL is becoming a useful OpenCL driver for AMD GPUs to run chipStar on.

**Comment by ArcherSmythe** [comment]
Thank you!
